src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/components/Legend.tsx

```diff
@@ -34,13 +67,6 @@ export default function Legend(props: IFormLegendProps) {
             ({getLanguageFromKey('general.optional', props.language)})
           </span>
         }
-        {props.helpText &&
```

Comment:
Hmm, I think we are missing a use case here.

With the new change, if you dont set `{help}` in the text resource, we now wont show help text at all.

So I think we will need some additional logic to ensure the old behavior is kept after all.

Reply:
Ok, I added some new code for handeling it.